styles.css

```diff
@@ -11,6 +10,26 @@ body {
   background: #f7f7f9;
 }
 
+.popupwindow {
+  display: none;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  opacity: 1;
+  overflow: scroll;
+  background: var(--neutrals-n-50, #c1c7d0);
+}
+
+.popupwindowhidden {
+  visibility: hidden;
+}
+
+.visible {
+  display: block;
+}
```

Comment:
- [ ] Good job designing your popup window 👏 , It could be better if it sticks to the figma design 😄 . Kindly center the popup window to meet the requirements ♻️ 

Your design 👍 

![Capture](https://github.com/enocol/MobileFirst/assets/80458199/89d32a05-00ad-431a-9c1b-f7f2fa7cd1ff)

Figma design

![Capture1](https://github.com/enocol/MobileFirst/assets/80458199/0ddab817-d4a7-4c3c-9146-e02296221682)